packages/contracts-bedrock/package.json

```diff
@@ -19,7 +19,7 @@
     "build": "hardhat compile && yarn build:ts && yarn typechain",
     "build:ts": "tsc -p tsconfig.json",
     "deploy": "hardhat deploy",
-    "test": "forge test",
+    "test": "yarn build:ts && forge test",
```

Comment:
This does slow things down a bit. Is it reasonable to expect ppl to run `yarn build` first?

Reply:
This is mostly for ci, running locally you just need to run `forge test` after building ts once. We can lower the fuzz runs in the local profile to speed it up